dev-python/uri_template/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@chymera.eu</email>
+		<name>Horea Christian</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+		<name>Python</name>
+	</maintainer>
```

Comment:
You should check with the Python project before you assign a package to them: https://wiki.gentoo.org/wiki/Project:Python#Adding_packages (but it's probably okay)

Reply:
> You should check with the Python project before you assign a package to them: https://wiki.gentoo.org/wiki/Project:Python#Adding_packages (but it's probably okay)

It's okay, the package is a direct dep of jsonschema which is already maintained by the Python project.